rpcs3/Emu/RSX/Overlays/overlay_list_view.cpp

```diff
@@ -40,8 +42,8 @@ namespace rsx
 			m_accept_btn->set_pos(30, height + 20);
 			m_cancel_btn->set_pos(180, height + 20);
 
-			m_accept_btn->set_text("Select");
-			m_cancel_btn->set_text("Cancel");
+			m_accept_btn->set_text(QCoreApplication::translate("RSXOverlay","Select").toStdString());
```

Comment:
Why not just add translation support in the set_text method itself and avoid all the code duplication?